.github/workflows/stryker.yml

```
@@ -0,0 +1,28 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: run Stryker - Node.js CI
+
+on:
+  push:
+    branches: [ master, stryker-action ]
```

Do you plan to keep that second branch name or is it jut for testing?

---

It was just for testing, it should be safe to do this before merging:

```suggestion
    branches: [ master ]
```